### PR TITLE
LPS-134089 Field groups must not be mapped to objects fields

### PR DESCRIPTION
**Pull request by mateussgn**
Related issue: https://issues.liferay.com/browse/LPS-134089

### CODE REVIEW

**Comment by mateussgn** [comment]
ci:test:sf

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:test:relevant

File: modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java [hunk 1]
```diff
@@ -230,25 +230,30 @@ private long _getObjectDefinitionId(
...
-			ddmStorageAdapterSaveRequest.getDDMFormValues();
+		for (DDMFormFieldValue ddmFormFieldValue : ddmFormFieldValues) {
+			String fieldType = ddmFormFieldValue.getType();
 
-		for (DDMFormFieldValue ddmFormValue :
-				ddmFormValues.getDDMFormFieldValues()) {
-
-			String objectFieldName = _getObjectFieldName(
-				ddmFormValue.getDDMFormField());
+			if (fieldType.equals("fieldset")) {
```
Comment (gabrielwas) [inline modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java:236]: Can you replace it with `StringUtil.equals` ?

**Comment by gabrielwas** [comment]
ci:test:sf

File: modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java (hunk 1 above)
Comment (mateussgn) [reply modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java]: Sure. Already did it, and replaced the string "fieldset" with the fieldset constant.

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:stop

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
Hey @gabrielwas, I sent directly to https://github.com/brianchandotcom/liferay-portal/pull/104739

**Comment by mateussgn** [comment]
ci:close

**Pull request closed without merging**